lib/routes/sciencedirect/current-issue.ts

```diff
@@ -76,7 +76,14 @@ async function handler(ctx) {
                     },
                 });
 
-                item.description = response.data[0].abstracts[0].html ?? '';
+                const abstracts = response.data?.[0]?.abstracts ?? [];
+                item.description =
+                    abstracts.length <= 1
+                        ? (abstracts[0]?.html ?? '')
+                        : abstracts
+                              .slice(0, 2)
```

Comment:
```suggestion
```
Maybe we can use everything from `abstracts` in case a new abstract section is added in the future.